architecture/core_objects/builds.adoc

```diff
@@ -166,6 +167,24 @@ Once you have the `*Secret*` created, you can add a `PushSecretName` field into
 `Output` section inside the `BuildConfig` and set it to the name of the
 `*Secret*` that you created, in this case `*dockerhub*`.
 
+For pulling the builder Docker image from private Docker registry, you have to specify
```

Comment:
Can you go ahead and add the sample for PushSecretName too?  you can make it one sample if you want.